wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -79,6 +79,9 @@ class WrongStringVisitor(base.BaseNodeVisitor):
         flags=re.X,  # flag to ignore comments and whitespaces.
     )
 
+    _valid_f_lookup_types = (ast.Str, ast.Num, ast.Name)
```

Comment:
This needs to be typed: `ClassVar[AnyNodes]`

And please, do not name things like `_f_` it is hard to read.

Reply:
Use `_formatted_` instead